src/actions/braze/test_braze.ts

```diff
@@ -0,0 +1,85 @@
+import * as chai from "chai"
+// import * as sinon from "sinon"
+// import concatStream = require("concat-stream")
+import * as Hub from "../../hub"
+
+import { BrazeAction } from "./braze"
+
+class BrazeActionTest extends BrazeAction {
+  name = "braze"
+
+  async form() {
+    const form = new Hub.ActionForm()
+    form.fields = [{
+      label: "Unique Key",
+      name: "braze_key",
+      required: true,
+      options: [
+        {name: "external_id", label: "external_id"},
+        {name: "braze_id", label: "braze_id"},
+      ],
+      type: "select",
+      default: "external_id",
+    }, {
+      label: "Export Label",
+      name: "braze_segment",
+      required: true,
+      type: "string",
+    }]
+    return form
+  }
+}
+
+const action = new BrazeActionTest()
+
+describe(`${action.constructor.name} unit tests`, () => {
+  describe("action", () => {
+    it("errors if there is no configuration for query", () => {
```

Comment:
I think we should have a test case for a successful action.

Reply:
added thanks